config/webpack.vendor.js

```diff
@@ -43,12 +43,9 @@ module.exports = [
    * Brace related
    */
   'brace',
-  'brace/mode/javascript',
   'brace/mode/ruby',
-  'brace/mode/html_ruby',
```

Comment:
Isn't this needed for erb?

Reply:
thats what i thought initially when i added it, when testing it seems like its not.